models/product/src/product-variant/fields-config.ts

```diff
@@ -27,7 +27,20 @@ export const restFieldsConfig: TModelFieldsConfig<TProductVariantRest> = {
 export const graphqlFieldsConfig: TModelFieldsConfig<TProductVariantGraphql> = {
   fields: {
     ...commonFieldsConfig,
-    attributesRaw: [],
+    attributesRaw: fake(() => [Attribute.random()]),
```

Comment:
Since we are populating the REST representation with one item in the `attributes` list, it makes sense to also have one in the GraphQL side.